rock_paper_scissors/rps.py

```diff
@@ -2,13 +2,25 @@
 
 import sys
 
+
 def rock_paper_scissors(n):
-  pass 
+    options = ['rock', 'paper', 'scissors']
```

Comment:
Good try! you could use recursion and make a find_result function inside of rps. 

def find_result(rounds_left, outcome=[]):
____if rounds_left == 0:
________result.append(outcome)
________return
____for option in options:
________find_result(rounds_left-1, outcome + [option])

find_result(n, [])
return result